File: Respostas/Bee_1117/1117.c
#include <stdio.h>

int main() {
	float AV1, AV2;
	
	do {
		scanf("%f", AV1);
		
		if ((AV1 < 0) || (AV1 > 10))
			printf("nota invalida\n");
	}
	while ((AV1< 0) || (AV1> 10));
	
	do {
		scanf("%f", AV2);
		
		if ((AV2 < 0) || (AV2 > 10))
			printf("nota invalida\n");
	}
	while ((AV2 < 0) || (AV2 > 10));
	
	printf("media = %.2f\n", (AV1 + AV2) / 2);
	
	return 0;
}

File: Respostas/Bee_1117/1117.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws IOException {
		double AV1, AV2;
		
		Scanner input = new Scanner(System.in);
		
		do {
			AV1 = input.nextDouble();
			
			if ((AV1 < 0) || (AV1 > 10))
				System.out.println("nota invalida");
		}
		while ((AV1 < 0) || (AV1 > 10));
		
		do {
			AV2 = input.nextDouble();
			
			if ((AV2 < 0) || (AV2 > 10))
				System.out.println("nota invalida");
		}
		while ((AV2 < 0) || (AV2 > 10));
		
		System.out.printf("media = %.2f\n", (AV1 + AV2) / 2);
	}
}